ydb/driver.py

```diff
@@ -23,10 +23,17 @@ class RPCCompression:
 def default_credentials(credentials=None, tracer=None):
     tracer = tracer if tracer is not None else tracing.Tracer(None)
     with tracer.trace("Driver.default_credentials") as ctx:
-        if credentials is not None:
+        if credentials is None:
+            ctx.trace({"credentials.anonymous": True})
+            return credentials_impl.AnonymousCredentials()
+        else:
             ctx.trace({"credentials.prepared": True})
             return credentials
 
+
+def credentials_from_env_variables(tracer=None):
+    tracer = tracer if tracer is not None else tracing.Tracer(None)
```

Comment:
you can write in short: `tracer or tracing.Tracer(None)..`

Reply:
good point, thanks.
now I simple move the code from other part. It will need replace all similar constructions in other commit.